controllers/tasks.js

```diff
@@ -339,7 +327,11 @@ const updateTaskStatus = async (req, res, next) => {
 
     if (task.taskNotFound) return res.boom.notFound("Task doesn't exist");
     if (task.notAssignedToYou) return res.boom.forbidden("This task is not assigned to you");
-    if (task.taskData.status === TASK_STATUS.VERIFIED || req.body.status === TASK_STATUS.MERGED)
+    if (
+      task.taskData.status === TASK_STATUS.VERIFIED ||
+      req.body.status === TASK_STATUS.MERGED ||
+      req.body.status === TASK_STATUS.BACKLOG
```

Comment:
NIT: we can destructure this `status` and use it here then

Reply:
Sure this will be done